docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: "3.9"
+
+services:
+  production:
+    image: todo-app:prod
+    env_file:
+      - env.list
```

Comment:
Why is this `env.list` rather than `.env`? Your Readme instructions still imply you expect `.env`